packages/shared/src/hooks/useDeleteSquad.ts

```diff
@@ -22,7 +22,7 @@ export const useDeleteSquad = ({
 }: UseDeleteSquadProps): UseDeleteSquadModal => {
```

Comment:
This whole part feels it could be part of useSquad, how did you feel about it looking at code?
(Don't need to refactor, just looking for your opinion as someone who touched the code)

Reply:
For me, the `useDeleteSquad` hook is more about showing a prompt confirmation when deleting a squad. So maybe we can rename it to `useDeleteSquadPrompt` instead. That way it should be clearer.

I'd rather refrain from refactoring the hook in this PR, since it's already touching a lot of code. But if you feel like we should have the functionality in useSquad hook, let's add a tech debt issue for that :)